Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -53,21 +52,20 @@ public UWP(Package package)
 
         private void InitializeAppInfo()
         {
-            AppxPackageHelper _helper = new AppxPackageHelper();
             var path = Path.Combine(Location, "AppxManifest.xml");
 
             var namespaces = XmlNamespaces(path);
             InitPackageVersion(namespaces);
 
             const uint noAttribute = 0x80;
-            const Stgm exclusiveRead = Stgm.Read | Stgm.ShareExclusive;
-            var hResult = SHCreateStreamOnFileEx(path, exclusiveRead, noAttribute, false, null, out IStream stream);
+            const Stgm nonExclusiveRead = Stgm.Read | Stgm.ShareDenyNone;
```

Comment:
I see the pr you referenced completely remove the share option. Is that the same as `Stgm.ShareDenyNone`?

Reply:
Yes, it's the default value.